src/app/shared/model/feeder-episode.model.ts

```diff
@@ -0,0 +1,68 @@
+import { Observable} from 'rxjs';
+import { HalDoc } from '../../core';
+import { BaseModel } from './base.model';
+
+export class FeederEpisodeModel extends BaseModel {
+  id: string;
+  guid: string;
+  authorName: string = '';
+  authorEmail: string = '';
+  enclosureUrl: string = '';
+
+  SETABLE = ['guid', 'authorName', 'authorEmail'];
+
+  constructor(private story: HalDoc, episode: HalDoc, loadRelated = true) {
+    super();
+    this.init(story, episode, loadRelated);
+  }
+
+  key() {
+    if (this.doc) {
+      return `prx.episode.${this.doc.id}`;
+    } else if (this.story) {
+      return `prx.episode.new.${this.story.id}`;
+    } else {
+      return null;
+    }
+  }
+
+  related() {
+    return {};
+  }
+
+  decode() {
+    // hmm, episode id is string but Haldoc id is number and making it number | string throws all kinds of errors
+    this.id = '' + this.doc['id'];
+    this.guid = this.doc['guid'];
+    // TODO: needs to default to account if not set,
+    //  should this be done here (with follow on story prx:account like with FeederPodcastModel)
+    //  or where we're using this model for the story?
```

Comment:
Is this defaulting to account name and email a model responsibility? Should get via this.story.follow('prx:account') like [FeederPodcastModel](https://github.com/PRX/publish.prx.org/blob/master/src/app/shared/model/feeder-podcast.model.ts#L61)?